### PR TITLE
Add new function for bbox world coords

### PR DESCRIPTION
**Pull request by lucabergamini**
Add utility function to get world coordinates of bboxes starting from `agents` array. 
Add tests for the function and use it in visualizer

### CODE REVIEW

File: l5kit/l5kit/rasterization/box_rasterizer.py [hunk 1]
```diff
@@ -25,6 +25,25 @@ def get_ego_as_agent(frame: np.ndarray) -> np.ndarray:  # TODO this can be usefu
...
+    corners_base_coords = (np.asarray([[-1, -1], [-1, 1], [1, 1], [1, -1]]) * 0.5)[None, :, :]
+
+    # compute the corner in world-space (start in origin, rotate and then translate)
+    corners_m = corners_base_coords * agents["extent"][:, None, :2]  # corners in zero
+    s = np.sin(agents["yaw"])
+    c = np.cos(agents["yaw"])
+    # note this is clockwise because it's right-multiplied and not left-multiplied later,
+    # and therefore we're still rotating counterclockwise.
+    rotation_m = np.moveaxis(np.array(((c, s), (-s, c))), 2, 0)
+    box_world_coords = corners_m @ rotation_m + agents["centroid"][:, None, :2]
```
Comment (perone) [inline l5kit/l5kit/rasterization/box_rasterizer.py:43]: I would add a comment here about what is the `[:, None, :2]` is doing or you can (even better) just add another variable with it, so you can name what is this term here.

**Review by perone — DISMISSED** [review]
Just a minor comment about adding a comment (no pun intended).

File: l5kit/l5kit/rasterization/box_rasterizer.py (hunk 1 above)
Comment (lucabergamini) [reply l5kit/l5kit/rasterization/box_rasterizer.py:46]: I've added comments for all the shape where `None` was involved :)

**Review by perone — APPROVED** [review]
Thanks !!

**Pull request merged**